### PR TITLE
release-drafter - Add category for translations

### PR DESCRIPTION
**Pull request by PabstMirror**
and remove `enhancement` so label to title 1-1 mapping

### CODE REVIEW

**Comment by jonpas** [comment]
Enhancement would make sense to go into separate IMPROVED header IMO. We can always manually group headers together for smaller releases.

File: .github/release-drafter.yml [hunk 1]
```diff
@@ -10,15 +10,16 @@ categories:
...
-      - 'area/translations'
   - title: '**FIXED:**'
     labels:
       - 'kind/bug fix'
   - title: '**CHANGED:**'
     labels:
-      - 'kind/enhancement'
       - 'kind/cleanup'
       - 'area/compatibility'
+  - title: '**Translations:**'
```
Comment (mharis001) [inline .github/release-drafter.yml]:
```suggestion
  - title: '**TRANSLATIONS:**'
```

**Comment by PabstMirror** [comment]
I think we can rename the existing labels,
I think it would be best if the labels are "What you see is what you get" to changelog categories

**Comment by jonpas** [comment]
I personally never liked current label system, much prefer the old way, like ACEX still has.

It doesn't have to be one-one in terms of nanes, but feature - added etc. would indeed nice.

**Pull request merged**